### PR TITLE
Improve esrpcodesign: Updated ESRP code sign configuration to improve perf

### PR DESCRIPTION
**Pull request by sharanya-rao**
[AB#212171](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/212171)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -54,11 +54,11 @@ steps:
   displayName: Sign exe, dll and powershell files
   inputs:
     ConnectedServiceName: 'CodeSigning-APEX'
-    FolderPath: '$(System.DefaultWorkingDirectory)'
+    FolderPath: '$(System.DefaultWorkingDirectory)/ECMA2Yaml'
```
Comment (superyyrrzz) [inline azure-pipelines.yml:57]: Just to confirm. I see you also added `ECMA2Yaml` at the start of some paths. Does the duplication reflect the folder structure correctly?

Comment (sharanya-rao) [reply azure-pipelines.yml:57]: yes - https://github.com/docascode/ECMA2Yaml/tree/develop/ECMA2Yaml/ECMA2Yaml

**Review by superyyrrzz — DISMISSED** [review]

File: CodeSign_Executable.json [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
       "MatchedPath": [
-        "*.ps1",
-        "*.exe",
-        "*.dll"
+        "ECMA2Yaml/bin/release/*.dll",
+        "ECMA2Yaml/bin/release/*.exe",
+        "IntellisenseFileGen/bin/Release/*.dll",
+        "IntellisenseFileGen/bin/Release/*.exe",
+        "ECMA2Yaml/Nuget/*.ps1",
+        "ECMAHelper/bin/Release/*.dll"
```
Comment (TianqiZhang) [inline CodeSign_Executable.json:11]:
```suggestion
        "ECMAHelper/bin/Release/netstandard2.0/*.dll"

```

File: CodeSign_Executable.json [hunk 3]
```diff
@@ -4,9 +4,12 @@
   "SignBatches": [
     {
       "MatchedPath": [
-        "*.ps1",
-        "*.exe",
-        "*.dll"
+        "ECMA2Yaml/bin/release/*.dll",
+        "ECMA2Yaml/bin/release/*.exe",
+        "IntellisenseFileGen/bin/Release/*.dll",
```
Comment (TianqiZhang) [inline CodeSign_Executable.json]: IntellisenseFileGen only needs .exe

**Review by TianqiZhang — APPROVED** [review]

**Pull request merged**